app/views/sport_sessions/index.html.erb

```diff
@@ -8,9 +8,15 @@
       <% end %>
     <% end %>
   </div>
-  <div data-map-list-toggle-target="map" class="d-none">
-    <!-- PUT THE MAP IN HERE -->
-    <p>Sorry, no sessions posted yet! And I am A Map. Post one here: <%= link_to "New Sport Session", new_sport_session_path %></p>
+  <!-- PUT THE MAP IN HERE -->
+  <div class="d-flex justify-content-center" data-map-list-toggle-target="map">
```

Comment:
I removed 1 div tag here, it was creating complexity and causing the issue and the child div was colapsing on itself.